app/src/components/BCIDView.tsx

```diff
@@ -30,6 +29,8 @@ const trustedLSBCCredentialIssuerRe =
 const redirectUrlTemplate = 'bcwallet://bcsc/v1/dids/<did>'
 const notBeforeDateTimeAsString = '2022-11-21T17:00:00.000Z'
 const connectionDelayInMs = 3000
+// const invitationId = '6cc22b56-fd0c-4b78-a7e4-c60c2e80e034'
+const invitationId = '0644a296-8be7-403c-95da-2bfb77ee95f1'
```

Comment:
This is fine for now but better to come from `invite.id` after calling `parseInvitation`